Orange/data/table.py

```diff
@@ -1065,10 +1065,20 @@ def has_missing(self):
         missing_x = not sp.issparse(self.X) and bn.anynan(self.X)   # do not check for sparse X
         return missing_x or bn.anynan(self._Y)
 
+    def has_missing_attribute(self):
+        """Return `True` if there are any missing attribute values."""
+        return not sp.issparse(self.X) and bn.anynan(self.X)    # do not check for sparse X
+
     def has_missing_class(self):
         """Return `True` if there are any missing class values."""
         return bn.anynan(self._Y)
 
+    def get_nan_frequency_attribute(self):
+        return np.isnan(self.X).sum() / self.X.size
```

Comment:
Sorry, I just noticed one more: could you return `0` if `self.X.size` is 0, to avoid division by zero? Same in the next function?